### ISSUE TITLE
Allow gunfire to pass over the Antimatter Bomb

### ISSUE THREAD

**Issue by SharkLordSatan** [HaloSpaceStation/HaloSpaceStation13#3548]
Currently, the Antimatter Bomb functions as a solid object that can't be shot over. On paper this is fine and dandy, but it kind of becomes an issue when you throw in guns that have object penetration into the mix.

Last round I watched a marine equipped with a DMR grab the bomb and drag it around while actively firing on the Covenant trying to get the bomb back. As the DMR has object-penetration apparently, this meant the marine could actively use the Antimatter Bomb as a mobile barricade, meaning that he could freely fire upon the Covenant, while the Covenant weren't able to shoot him as none of their weapons had object-penetration qualities (unless they managed to flank him).

Basically what I'm asking is that we should make it so that the Antimatter Bomb doesn't block gunfire going over it, so that we can prevent people from trying to cheese it and utilize it as a barricade both when planted and unplanted.

**Comment by X0-11** [comment]
This has been implemented
